src/Equinor.ProCoSys.IPO.Command/InvitationCommands/UnAcceptPunchOut/UnAcceptPunchOutCommand.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using MediatR;
+using ServiceResult;
+
+namespace Equinor.ProCoSys.IPO.Command.InvitationCommands.UnAcceptPunchOut
+{
+    public class UnAcceptPunchOutCommand : IRequest<Result<string>>, IInvitationCommandRequest
```

Comment:
Add tests in AccessValidatorTests
(Goes for other commands too, as mentioned in prev PR)